### PR TITLE
Updated italian translations

### PR DESCRIPTION
**Pull request by JohnKiller**
Hi!

It's been a looong time, but I had some free time and I've updated the translations with the new strings.

Thanks again for your amazing program.

### CODE REVIEW

File: website/it.js [hunk 1]
```diff
@@ -116,21 +116,21 @@ jquery_lang_js.prototype.lang.it = {
 		'Scrivi i tuoi commenti sulla <a href="http://unchecky.userecho.com/">pagina UserEcho</a>',
 	
 	'How do I uninstall Unchecky?':
-		'',
+		'Come faccio a disinstallare Unchecky?:',
```
Comment (RaMMicHaeL) [inline website/it.js]: Is the `:` symbol at the end a typo?

Comment (JohnKiller) [reply website/it.js]: Yep, it's a typo

Comment (RaMMicHaeL) [reply website/it.js]: 👍 I pushed a fix

**Comment by RaMMicHaeL** [comment]
Thank you for the translation update, it will be included in the next version of Unchecky.

**Pull request merged**